### DISCUSSION TITLE
Failed to initialize OpenGL loader

### DISCUSSION THREAD

**Discussion by Karnatour** [ocornut/imgui#7160] [New Users Build/Link/Run issues ONLY!]
Hi i have question will imgui work if i have opengl version 4.6.0 with compability profile ? or do i need to force version 3.3. Thanks

**Comment by ocornut** [comment]
It should work yes. 
Try to step in `imgl3wInit()` and specify OS/details.

**Comment by Karnatour** [comment]
It was error in my program, works with 4.6.0. Thanks and merry Christmas
